### PR TITLE
Add terraform-import machinepool test case 

### PR DESCRIPTION
**Pull request by prabinovRedhat**
Based on this test case: https://polarion.engineering.redhat.com/polarion/#/project/OSE/workitem?id=OCP-66403
Screenshot 2023-12-26 at 16 54 54

### CODE REVIEW

**Comment by prabinovRedhat** [comment]
cc: @amalykhi @xueli181114

File: tests/e2e/machine_pool_test.go [hunk 1]
```diff
@@ -651,6 +651,64 @@ var _ = Describe("TF Test", func() {
...
 		})
 	})
+
+	Describe("Validate terraform Import operations", func() {
+		var mpService *exe.MachinePoolService
+		var importService exe.ImportService
+
+		BeforeEach(func() {
+			mpService = exe.NewMachinePoolService(con.MachinePoolDir)
+			importService = *exe.NewImportService(con.ImportResourceDir) // init new import service
```
Comment (xueli181114) [inline tests/e2e/machine_pool_test.go:661]: How about import service destroy in AfterEach? What will happen when destroy an imported resource?

**Comment by prabinovRedhat** [comment]
Screenshot 2024-01-03 at 14 31 37

File: tests/e2e/machine_pool_test.go (hunk 1 above)
Comment (prabinovRedhat) [reply tests/e2e/machine_pool_test.go:625]: I fixed

**Comment by prabinovRedhat** [comment]
@xueli181114 @shellymiron @amalykhi  please review

**Pull request merged**